string node_name         
bool   is_healthy          
string info             
bool   is_unrecoverable_error
bool   is_just_started
bool   is_busy